cmd/internal/flag.go

```diff
@@ -194,3 +195,10 @@ func PolicyExceptionEnabled() bool {
 func LeaderElectionRetryPeriod() time.Duration {
 	return leaderElectionRetryPeriod
 }
+
+func printFlagSettings(logger logr.Logger) {
```

Comment:
This function will print all flags, even those not explicitly passed with their (default) values?

Reply:
Yes.

Reply:
Ok good